modules/ocf_mirrors/templates/record-mirrors-stats.py.erb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+import os
+import pymysql
+from datetime import date
+
+OCFSTATS_PWD = "<%= @ocfstats_password %>"
+MIRRORS_PATH = '/opt/mirrors/ftp'
+LOG_PATH = '/var/log/apache2'
+LOG_NAME = 'mirrors.ocf.berkeley.edu_access.log'
+
+sources = [mirrored for mirrored in os.listdir(MIRRORS_PATH)
+           if os.path.isdir(os.path.join(MIRRORS_PATH, mirrored)) and not mirrored.startswith('.')]
+
+sources.append('other') # catchall
```

Comment:
2 spaces before inline comments per PEP8: https://www.python.org/dev/peps/pep-0008/#inline-comments

pre-commit isn't enforcing this?